src/views/users/User/UserCreateUpdate.vue

```diff
@@ -233,9 +235,14 @@ export default {
     disableMFAFieldIfNeed(user) {
       // SECURITY_MFA_AUTH 0 不开启 1 全局开启 2 管理员开启
       const adminUserIsNeed = (user?.is_superuser || user?.is_org_admin) && this.$route.meta.action === 'update' &&
-        store.getters.publicSettings['SECURITY_MFA_AUTH'] === 2
-      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 1 || adminUserIsNeed) {
-        this.fieldsMeta['mfa_level'].disabled = true
+        store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.onlyAdminUsers
+      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.allUsers) {
+        this.fieldsMeta['mfa_level'].options = [{ 'value': 3, 'label': this.$t('MFAAllUsers') }]
+        this.initial.mfa_level = MFALevel.allUsers
+      }
+      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.onlyAdminUsers && adminUserIsNeed) {
+        this.fieldsMeta['mfa_level'].options = [{ 'value': 4, 'label': this.$t('MFAOnlyAdminUsers') }]
+        this.initial.mfa_level = MFALevel.onlyAdminUsers
       }
     }
   }
```

Comment:
No issues found. The code appears to be well structured without any known errors, irregularities, or improvements needed. It's a minimal Vue component for MFA settings in a web application. If you have specific concerns about its functionality or use case that I should take into account, please mention them!